### PR TITLE
Link to existing docs and drop duplicates.

### PR DESCRIPTION
**Pull request by jmcarp**
We currently document deploying apps built with django, rails, and a few other frameworks. Most of this documentation can be found elsewhere, so let's link to the official buildpack docs and sample applications instead of writing our own. To start with, I cut out lots of framework-specific docs and replaced them with upstream links. Let me know if I took out too much or didn't say enough!

### CODE REVIEW

**Comment by brittag** [comment]
Related issue about these guides being a lil bit rusty right now: https://github.com/18F/cg-site/issues/560

**Comment by afeld** [comment]
I am 💯  for using upstream docs instead of our own, but also don't want to lose any important information that isn't reflected elsewhere (yet). I haven't gone through line-by-line to check... @jmcarp how confident are you about it?

**Comment by jmcarp** [comment]
To clarify, I don't think that the upstream buildpack docs cover every line of these docs. I don't think upstream describes meteor. That's fine! It's beyond the scope of cloud.gov (and beyond our competence) to document every language and framework that runs on the platform.

In some cases, the heroku docs are more detailed than the cf docs, so we can link to those too.

**Comment by sharms** [comment]
Is there any evidence to suggest that a developer getting started in Meteor (not a core 18F framework) would look to the Cloud.gov team for direction in that area?

**Comment by jmcarp** [comment]
@afeld: would you prefer breaking this up into smaller patches? For example, we could start by replacing the meteor docs with a link to the unsupported meteor buildpack, or just delete them entirely since they're unsupported, and go from there?

File: content/docs/apps/frameworks.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+menu:
+  docs:
+    parent: advanced
```
Comment (afeld) [inline content/docs/apps/frameworks.md]: Thoughts about using `parent: apps` instead?

**Comment by afeld** [comment]
It's ok, we can punt on it: https://github.com/18F/cg-site/issues/162. Just one more thing: Thoughts about adding [aliases](https://gohugo.io/extras/aliases/) for the old URLs to redirect to the Frameworks page?

/cc https://github.com/cloudfoundry/docs-buildpacks/pull/114

**Review by afeld — APPROVED** [review]

**Pull request merged**